recipeapp/src/Recipes/index.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import SearchBar from './SearchBar/SearchBar';
```

Comment:
Assuming the change to the `SearchBar` directory structure as stated above, this line could become
```JavaScript
import SearchBar from './SearchBar';
```

Reply:
Changes made.